test/lib/ansible_test/_data/sanity/code-smell/parsable-yaml.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import re
+import sys
+
+from yaml import load
+from yaml.error import YAMLError
+
+# TODO: We want CSafeLoader because it is more strict than SafeLoader.  But should we run this test
+# with reduced accuracy when we don't have CSafeLoader?
+from yaml import CSafeLoader as SafeLoader
```

Comment:
I believe there is a requirements.txt.file for sanity, so we can list the best module there.

Reply:
I don't think that will work because both of these are the same pypi package (pyyaml).  What happens is that when the package sdist is downloaded, the python module gets built and installed.  If the compilation chain to build the C extension exists (gcc, the libyaml header files, etc) then the we get yaml.CSafeLoader.  If the compilation chain is incomplete, then we only get the pure-python version.  (If there was an anylinux wheel, then we could just download that.  but there are only windows wheels, not an anylinux wheel: https://pypi.org/project/PyYAML/#files )

We could also be getting pyyaml from a system package in which case they probably do have the C extension built.  (But that's not something that we can specify in the requirements.txt)